test/jdk/ProblemList.txt

```diff
@@ -533,6 +533,8 @@ java/beans/XMLEncoder/Test6570354.java 8015593 macosx-all
 
 java/foreign/TestMismatch.java 8249684 macosx-all
 java/foreign/TestUpcallStack.java 8275584 macosx-aarch64
+java/foreign/TestIntrinsics.java 8275584 macosx-aarch64
```

Comment:
What happened to this test? This seems to be working today

Reply:
Yeah, that one should work I think. Let me try to re-add it.